src/lib/configs/project/fluence.ts

```diff
@@ -159,6 +160,7 @@ const serviceSchema: JSONSchemaType<ServiceV1> = {
     },
   },
   required: ["get"],
+  additionalProperties: false,
```

Comment:
Is there are lib to generate JSONSchema from decorator annotations on an runtime object itself?
Would benefit bcs that would keep object strucure in a single place

Reply:
not sure about that. I would only say that it's most flexible to write the schema by hand using the schema syntax itself without any additional wrappers/tools. This way a lot of things can be controled and some of the things can be shared easilly between config versions. Also typesacript helps to write this schemas a bit. It is not the best experience to use this schema lib for me and to maintain all that, I will tell you that. But I find it's worth it because users can enjoy good autocomplete validation and docs for configs right inside their IDE